src/main/java/pages/TestTypeCategoryPage.java

```diff
@@ -55,7 +85,27 @@ public void selectTestTypeFromListByXpath(String testType) {
         }
     }
 
+    public void selectSuggestedTestTypeFromListByXpath(String testType) {
+        logger.debug("Page source after wait 2: " + getDriver().getPageSource());
+        List<WebElement> webElementList = findElementsByXpath(PAGE_ALL_BUTTON_XPATH);
```

Comment:
You can probably do some fancy thing with XPath instead of looping through the list of nodes you get back
- https://sodocumentation.net/xpath/topic/3096/find-nodes-that-have-a-specific-attribute 
- my XPath skills aren't so hot so this might not be possible!

Reply:
ok thanks Dan, will see if we can maybe add this in when we refactor the pack :) Thanks for reviewing!